src/js/contexts/AnnounceContext/announce-context.stories.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { grommet } from 'grommet/themes';
+import { Box, Grommet, Text, AnnounceContext } from 'grommet';
+
+class Announcer extends Component {
+  static propTypes = {
+    announce: PropTypes.func.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    const { message, role, politenessSetting } = this.props;
+    this.state = {
+      message:
+        message || 'Here is a simple announcement. This will soon disappear',
+      politenessSetting: politenessSetting || 'polite',
+      role: role || 'log',
+    };
+  }
+
+  componentDidMount() {
+    const { announce } = this.props;
+    const { message, politenessSetting } = this.state;
+    const timeout = 3000;
+    announce(message, politenessSetting, timeout);
+  }
+
+  render() {
+    const { message, politenessSetting, role } = this.state;
+    return (
+      <Text align="center" role={role} aria-live={politenessSetting}>
+        {message}
+      </Text>
+    );
+  }
+}
+
+const AnnounceContextComponent = props => (
+  <Grommet theme={grommet} full>
+    <Box
+      justify="center"
+      align="center"
+      pad="xlarge"
+      background="linear-gradient(102.77deg, #865ED6 -9.18%, #18BAB9 209.09%)"
```

Comment:
I think this level of extra detail can be a little distracting, as the reader is drawn to it and away from the primary focus of the AnnounceContext. Is it needed?

Reply:
Addressed in recent commit. Thanks !!